invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/communities.js

```diff
@@ -50,67 +48,35 @@ function ResultsGridItemTemplate({ result, index }) {
 }
 
 export function CommunitiesResultsItemTemplate({ result, index }) {
-  const community_type = _get(
-    result,
-    "ui.type.title_l10n",
-    i18next.t("No community type")
-  );
+  const community_type = result.ui?.type?.title_l10n;
+  const visibility = result.access.visibility;
+  let visibilityColor = "red";
+  let visibilityText = i18next.t("Restricted");
+  let visibilityIcon = "ban";
+  if (visibility === "public") {
+    visibilityColor = "green";
+    visibilityText = i18next.t("Public");
+    visibilityIcon = undefined;
+  }
   return (
-    <Item key={index}>
-      <Image
-        wrapped
-        src={result.links.logo}
-        fallbackSrc="/static/images/square-placeholder.png"
-        size="tiny"
-        className="community-logo"
+    <>
+      <ComputerTabletCommunitiesItem
```

Comment:
Why are two components needed? Do Mobile and Computer layout change that much?
Did you consider using composition or another  pattern that allowed some reusability?